apps/go-mod-validator/jest.config.ts

```diff
@@ -0,0 +1,12 @@
+import { Config } from "jest";
```

Comment:
Whoops I thought my commit deleted this - please remove.

Reply:
Done